examples/calendar/backend/services/users/controller.ts

```diff
@@ -0,0 +1,39 @@
+import { NextFunction, Request, Response } from "express";
+import { User } from "../../models/user.model";
+import { Logger } from "tslog";
+
+
+const log: Logger = new Logger({ name: "errorLogger" });
+
+export const find = async (req: Request, res: Response, next: NextFunction) => {
+	if (!(req as any).query.hasOwnProperty("publicAddress")) {
+		return res.status(401).send({ error: "Please send Address!" });
+	}
+	return res.status(200).send(await User.findAll({where: { publicAddress: req.query.publicAddress }}));
+};
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+	if (!(req as any).hasOwnProperty("user")) {
+		return res.status(401).send({ error: "No User found!" });
+	}
+	return User.findByPk((req as any).user.payload.id)
```

Comment:
what happens if payload is undefined?

Reply:
Added if condition to ensure payload is not undefined

Reply:
Can you please check that the code was pushed?